src/Bedrock.Framework.Experimental/Protocols/RabbitMQ/FrameType.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Bedrock.Framework.Experimental.Protocols.RabbitMQ
+{
+    internal enum FrameType : byte
+    {
+        Method = 1,
+        Header = 2,
+        Body = 3,
+        HeartBeat = 8,
+        End = 206
+    }
+}
```

Comment:
Nit: newline

Reply:
where newline is missing ?

Reply:
At L11 the file ends without newline. Indicated by 
![grafik](https://user-images.githubusercontent.com/5755834/76630691-bd82ce00-6540-11ea-8ac7-7cd2cfbd77ee.png)
and to be consistent with other files in the project.

Reply:
ok, fixed.